src/server/player_sao.cpp

```diff
@@ -685,6 +687,8 @@ bool PlayerSAO::checkMovementCheat()
 
 	player_max_walk = MYMAX(player_max_walk, override_max_H);
 
+	player_max_walk *= anticheat_movement_tolerance;
```

Comment:
wrong place to apply this IMO.
it should be a factor of `required_time`

Reply:
> wrong place to apply this IMO. it should be a factor of `required_time`

I tested it with `required_time` and player just can't move(resetting position) if they cheated once

Reply:
> > wrong place to apply this IMO. it should be a factor of `required_time`
> 
> I tested it with `required_time` and player just can't move anymore(resetting position on the slightest move) if they cheated once

Nevermind, my bad, it needed dividing instead of multiplying